client/landing/stepper/utils/shuffleArray.ts

```diff
@@ -0,0 +1,36 @@
+const SESSION_STORAGE_KEY = 'goals-step-seed';
+
+let localSeed: string | null = null;
+
+const getSeed = () => {
```

Comment:
@p-jackson, here's our seed function. If that fails, it tries fetching/saving to session storage and falls back to a local variable. Let me know what you think.

Reply:
Yes this looks like the right approach to me 👍 
The local variable will be faster so you could switch the logic to `localSeed ?? sessionStorage.getItem( ... )`? Use the local variable like a cache?

I wasn't expecting the randomisation function to look like `Math.sin( ++seed ) * 10000` 😅 but TIL that native JavaScript doesn't support the ability to provide your own seed! I'd suggested it just assuming it was a built-in like every other language 🤷

ChatGPT suggested a function like this, which would provide less regular results than `sin()`
```
function seededRandom(seed) {
  let value = seed;
  return function() {
    value = (value * 9301 + 49297) % 233280;
    return value / 233280;
  };
}
```